src/main/java/bobby/ui/Bobby.java

```diff
@@ -0,0 +1,366 @@
+        try {
+            LocalDateTime by = LocalDateTime.parse(details[1].trim(), formatter);
+            Deadlines currDeadline = new Deadlines(details[0].trim(), by);
+            taskList.add_task(currDeadline);
+            System.out.println("Got it. I've added this task:\n"
+                    + currDeadline.toString() + "\n"
+                    + "Now you have " + taskList.getSize() + " tasks in the list.\n");
+            check_action(parser.getActionType(scan.next()));
+        } catch (DateTimeParseException e) {
+            System.out.println("OOPS!!! Format of deadline should be "
+                    + "(deadline xxx /by yyyy-MM-dd HHmm)\n");
+            check_action(parser.getActionType(scan.next()));
+        }
+    }
+
+    /**
+     * Handles the "event" command by adding a new Event task to the task list.
+     *
+     * @throws Exception if there is an issue processing the next action.
+     */
+    private static void handleEvent() throws Exception {
+        String input = scan.nextLine();
+        String details[] = input.split("/from");
+        if (details.length < 2 || details[0].trim().isEmpty()) {
+            System.out.println("OOPS!!! Format of events should be "
+                    + "(event xxx /from yyyy-MM-dd HHmm /to yyyy-MM-dd HHmm)\n");
+            check_action(parser.getActionType(scan.next()));
+            return;
+        }
+        String duration[] = details[1].split("/to");
+        if (duration.length < 2 || duration[0].trim().isEmpty()
+                || duration[1].trim().isEmpty()) {
+            System.out.println("OOPS!!! Format of events should be "
+                    + "(event xxx /from yyyy-MM-dd HHmm /to yyyy-MM-dd HHmm)\n");
+            check_action(parser.getActionType(scan.next()));
+            return;
+        }
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        try {
+            LocalDateTime from = LocalDateTime.parse(duration[0].trim(), formatter);
+            LocalDateTime to = LocalDateTime.parse(duration[1].trim(), formatter);
+            Events currEvent = new Events(details[0].trim(), from, to);
+            taskList.add_task(currEvent);
+            System.out.println("Got it. I've added this task:\n"
+                    + currEvent.toString() + "\n"
+                    + "Now you have " + taskList.getSize() + " tasks in the list.\n");
+            check_action(parser.getActionType(scan.next()));
+        } catch (DateTimeParseException e) {
+            System.out.println("OOPS!!! Format of event should be "
+                    + "(event xxx /from yyyy-MM-dd HHmm /to yyyy-MM-dd HHmm)\n");
+            check_action(parser.getActionType(scan.next()));
+        }
+    }
+
+    /**
+     * Determines and executes the action based on user input.
+     *
+     * @param action The type of action to perform, as determined by user input.
+     */
+    private static void check_action(ActionType action) throws Exception {
```

Comment:
Perhaps this could be camelcase to be more in line with naming conventions for methods? Something like checkAction